symantec_endpoint_security/help.md

```diff
@@ -37,6 +37,44 @@ Example input:
 
 ### Actions
 
+#### Blacklist
+
+This action is used to blacklist MD5 or SHA256 hashes.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|description|string|None|True|Description for the blacklist|None|Hashes banned via InsightConnect|
+|domain_id|string|None|False|ID of the domain to apply the blacklist to. Omitting this input will apply the blacklist to all domains (globally)|None|0AF740760A0414711FAA4F8BD5293158|
+|hashes|[]string|None|True|Hashes (MD5 or SHA256) to add to the blacklist. Note: only one type of hash is allowed at a time|None|["9de5069c5afe602b2ea0a04b66beb2c0"]|
+|name|string|None|True|Name for the blacklist|None|Insightconnect Blacklist|
+
+Example input:
+
+```
+{
+  "description": "Hashes banned via InsightConnect",
+  "domain_id": "0AF740760A0414711FAA4F8BD5293158",
+  "hashes": ["9de5069c5afe602b2ea0a04b66beb2c0"],
+  "name": "Insightconnect Blacklist"
```

Comment:
```suggestion
  "name": "Hash Blacklisted from InsightConnect"
```

Reply:
This doesn't fit, the action allows for blacklisting multiple hashes into a single list - this "name" property is the name of the list. Are you okay with "Hashes Blacklisted from InsightConnect"?

Reply:
Yeah, Hashes.. looks good 👍